### ISSUE TITLE
export statements about Sinopia resources pointed to by your resource

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#2187]
to support re-use of data / cooperative cataloging, an institution could create a resource that points to another institution's resource, for example, Stanford creates an Instance, and says it's an instanceOf a Work created by Penn.

currently when we export resources, we only export resources in an institution's own "group". So in the above example, Stanford would have information about its Instance, and a statement that it's an instance of Yale's Work, but wouldn't have any information about the Yale Work.

Stanford would like those statements about the Yale Work to be included in the Stanford export.

open question: if that Yale Work then references other Works, does that get included too?

**Comment by michelleif** [comment]
@jermnelson when you export via API, do we export any other statements referred to by resource in question?

**Comment by jermnelson** [comment]
> @jermnelson when you export via API, do we export any other statements referred to by resource in question?

No, when accessing a resource via the API, other resources that may be referenced in the RDF are not exported.
